src/popup/utils/utils.ts

```diff
@@ -727,3 +727,9 @@ export function connectFrames(sdk: ISdk | AeSdkWallet) {
     handleUnknownError(error);
   }
 }
+
+export function checkImageAvailability(url: string): Promise<boolean> {
```

Comment:
I suggest to add a comment that describes why we have to use custom image existence check instead of the previous solution (problems on iOS).

Reply:
Your comment actually didn't describe why this function exists. I suggest to add an info describing that we need to check for the image existence because regular image error catching didn't work on iOS.